week-3-practise/exercise-3/exercise-3.js

```diff
@@ -0,0 +1,29 @@
+// you are given two array names and teamName
+//write a function to check if the right team name is selected.
+//hint use array.every method
+
+var names = [
+  "David",
+  "Julio",
+  "Marcathy",
+  "Sammy",
+  "Deadly",
+  "carmel",
+  "Esus",
+  "Damina"
+];
+var teamName = ["Frank", "Junior", "Smart", "Gene", "Marcus", "zack", "Fabio"];
+
+var teamOnlyList = names.every(name => teamName.includes(name) !== true); // complete this statement
```

Comment:
This is correct, but you can make the code a bit shorter. Remember that `.includes()` always returns a boolean. So you don't need to compare against `true`.